AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -176,6 +176,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60823
 kotp1000000.xyz,blog.donia-tech.net#%#//scriptlet("adjust-setInterval", "#timer", "", "0.02")
 kotp1000000.xyz,blog.donia-tech.net#%#AG_onLoad(function(){if(document.querySelector("#timer")){var b=new MutationObserver(function(){var a=document.querySelector("#place > a[href]");a&&(window.location=a.href);a&&(b.disconnect(),a.remove())});b.observe(document,{childList:!0,subtree:!0})}});
+! https://github.com/AdguardTeam/AdguardFilters/issues/60877
+me.fo#%#//scriptlet('prevent-addEventListener', 'scroll')
```

Comment:
Too wide rule. Better something like this `me.fo#%#//scriptlet('prevent-addEventListener', 'scroll', '/function\([a-z]?\){return"undefined"/')`